tooling/hcpctl/cmd/release/options.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2025 Microsoft Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package release
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/cli"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/Azure/ARO-HCP/tooling/hcpctl/pkg/version"
+)
+
+func DefaultReleaseStatusOptions() *RawReleaseStatusOptions {
+	return &RawReleaseStatusOptions{
+		OutputFormat: "yaml",
+	}
+}
+
+func BindReleaseStatusOptions(opts *RawReleaseStatusOptions, cmd *cobra.Command) error {
```

Comment:
Making Bind a function on the `opts` follows the kubectl and oc patterns better.